app/models/concerns/created_by.rb

```diff
@@ -0,0 +1,16 @@
+module CreatedBy
```

Comment:
sometimes I think everything needs to be namespaced and sometimes I don't, there are pros and cons - since we control every project it's never been a problem with conflicts - currently the codebase is inconsistent so we should come up with a pattern/policy